src/main/java/com/telefonica/iot/cygnus/sinks/OrionHDFSSink.java

```diff
@@ -90,7 +90,7 @@
     private String krb5LoginConfFile;
     private String krb5ConfFile;
     private boolean serviceAsNamespace;
-    private HDFSBackend persistenceBackend;
+    private HDFSBackendImpl persistenceBackend;
```

Comment:
The same thing goes to the class where this update has been included. I mean, it should be OrionHdfsSink instead of OrionHDFSSink, right?